lib/workers/repository/updates/generate.ts

```diff
@@ -325,6 +325,24 @@ export function generateBranchConfig(
       )
     ),
   ].filter(is.nonEmptyString);
+  // combine excludeCommitPaths for multiple manager experience
+  const hasExcludeCommitPaths = config.upgrades.some(
+    (u) => u.excludeCommitPaths && u.excludeCommitPaths.length > 0
+  );
+  if (hasExcludeCommitPaths) {
+    config.excludeCommitPaths = Object.keys(
+      config.upgrades.reduce((acc: Record<string, boolean>, upgrade) => {
+        if (upgrade.excludeCommitPaths) {
+          upgrade.excludeCommitPaths.forEach((p) => {
+            acc[p] = true;
+          });
+        }
+
+        return acc;
+      }, {} as Record<string, boolean>)
+    );
+  }
```

Comment:
I think this is the wrong way, normally a config option needs to be mergable, then config merged will do this automagically.

@rarkins should we make `excludeCommitPaths` mergable?
![image](https://user-images.githubusercontent.com/1798109/180146681-26a6e1da-f148-4011-bcfc-f99741188a00.png)

Reply:
Even if mergeable I didn't think we did it that way. Generally we either:
- Take the value of updates[0], or
- Do a context-aware merge (e.g. make sure every update is automerging, etc)